src/patternfly/layouts/Page/styles.scss

```diff
@@ -1,110 +1,307 @@
+  --pf-l-page__main-nav--BackgroundColor:                   var(--pf-global--BackgroundColor--light-100);
+
+  // Main section modifiers
+  --pf-l-page__main-section--m-light--BackgroundColor:      var(--pf-global--BackgroundColor--light-100);
+  --pf-l-page__main-section--m-dark--BackgroundColor:       var(--pf-global--BackgroundColor--dark-transparent-200);
+
+  /* stylelint-enable */
+
+  // Base
   display: grid;
   min-height: 100vh;
-  padding: 0;
-  margin: 0;
-  grid-template-columns: var(--pf-l-page__sidebar--md--Width) 1fr;
-  grid-template-rows: auto 1fr;
+  grid-template-columns: 1fr;
+  grid-template-rows: max-content 1fr;
   grid-template-areas:
-    "header header"
-    "main main";
+    "header"
+    "main";
 
   @media (min-width: $pf-global--breakpoint--md) {
-    &.pf-m-expanded {
-      grid-template-areas:
-        "header header"
-        "sidebar main";
-    }
-  }
-
-  &__sidebar {
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    width: var(--pf-l-page__sidebar--sm--Width);
-    grid-area: sidebar;
-    background-color: var(--pf-l-page__sidebar--BackgroundColor);
-    box-shadow: var(--pf-l-page__main-section--BoxShadow);
-    transform: translate3d(-100%, 0, 0);
-
-    .pf-m-expanded & {
-      transform: translate3d(0, 0, 0);
-    }
-
-    @media (min-width: $pf-global--breakpoint--md) {
-      position: static;
-      display: none;
-      width: 100%;
-      .pf-m-expanded & {
-        display: block;
-      }
-    }
+    grid-template-columns: max-content 1fr;
+    grid-template-areas:
+      "header header"
+      "nav main";
   }
 
+  // Header
   &__header {
-    // force the container follow the dark theme
+    // Force the container follow the dark theme
     @extend %pf-t-dark;
 
     grid-area: header;
     display: flex;
     flex-wrap: wrap;
+    align-items: flex-end;
+    max-width: 100%;
+    min-height: var(--pf-l-page__header--MinHeight);
+    padding-right: var(--pf-l-page__header--PaddingRight);
+    padding-left: var(--pf-l-page__header--PaddingLeft);
+    overflow-x: hidden;
 
     > * {
       display: flex;
       align-items: flex-end;
     }
 
+    // Toggle
+    &-sidebar-toggle {
+      padding: var(--pf-l-page__header-sidebar-toggle--Padding);
+      // Ensure button does not adjust positioning
+      margin-right: var(--pf-l-page__header-brand-link--MarginLeft);
+      margin-left: calc(var(--pf-l-page__header-sidebar-toggle--Padding) * -1);
+      font-size: var(--pf-l-page__header-sidebar-toggle--FontSize);
+      line-height: 1;
+      border: none;
+    }
+
+    // Brand
     &-brand {
-      flex: 0 0 var(--pf-l-page__sidebar--md--Width);
+      align-items: flex-end;
+      padding-top: var(--pf-l-page__header-brand--PaddingTop);
+      padding-bottom: var(--pf-l-page__header-brand--PaddingBottom);
+
+      @media (min-width: $pf-global--breakpoint--md) {
+        flex: 0 0 var(--pf-l-page__header-brand--Width--lg);
```

Comment:
Same as a comment I left above - should this have it's own `FlexBasis` var that is set to `var(--pf-l-page__header-brand--Width--lg)`?